app/themes/user-admin/e2e/user-edit/user-edit.e2e-spec.ts

```diff
@@ -0,0 +1,102 @@
+import { UserAdminPage } from '../app.po';
+import { protractor, browser, element, by, WebElement } from 'protractor';
+
+describe('User Edit Page', function() {
+    let page: UserAdminPage;
+
+    beforeEach(() => {
+        page = new UserAdminPage();
+    });
+
+    it('should login successfully with right email and password', () => {
+        page.login();
+    });
+
+    it('should lead to User List page at click of the User List link', () => {
+        //Click UserList in Nav to get to UserList Page
+        let userListNav = browser.findElement(protractor.By.cssContainingText('nav a', 'User List'));
+        userListNav.click();
+    });
+
+    //this test should lead to the user created inside the user-create e2e test    
+    it('should lead to individual user edit page at click of EDIT', () => {
+
+        let edit = browser.findElement(protractor.By.css('tbody tr:first-child a'));
```

Comment:
I see this notice when i run the tests: `[22:18:28] W/element - more than one element found for locator By(css selector, tbody tr:first-child td) - the first result will be used`